_layouts/base.html

```diff
@@ -116,14 +116,24 @@ <h4 style="padding: 0.5rem 1rem; margin-bottom: 0">Web and design team practices
             </ul>
           </nav>
           <script>
-            // Toggle mobile sidebar nav
-            var toggle = document.querySelector('.p-sidebar__toggle');
-            var sidebarContent = document.querySelector('.p-sidebar__content');
+            document.addEventListener("DOMContentLoaded", function(event) {
```

Comment:
I deliberately didn't wrap this in `DOMContentLoaded`, because its placement in the markup is directly after the `<nav>` in question - this leads to the functionality being available literally as soon as possible - as in, as soon as the browser has parsed the `<nav>` section, before the rest of the DOM has loaded.

You should use the `DOMContentLoaded` event iff the `<script>` block was instead included in the `<head>`. Or you could move the script block to the bottom of the document, in which case you'd never need `DOMContentLoaded`. I chose this particular pattern, but I'm happy for you to argue for another. However, if you're going to use `DOMContentLoaded`, this script block should be moved.

Reply:
I honestly didn't give it much thought at all, and now that I have it definitely feels unnecessary.

Removed